modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss

```diff
@@ -84,29 +70,43 @@
 		background-repeat: repeat-x;
 		background-size: 6px 1px;
 		bottom: 0;
-		box-sizing: border-box;
 		content: '';
 		height: 1px;
 		left: 0;
 		position: absolute;
 		right: 0;
 	}
 
-	.box {
-		max-height: 0;
-		opacity: 0;
-		transition: all 0.5s;
+	.layout-show-details {
+		background: #f2f6ff;
 	}
 
-	.box.show-box {
-		max-height: 300px;
-		opacity: 1;
-		transition: all 0.5s;
+	.policy-detail-title::after {
+		background-color: var(--primary);
+		content: '';
+		display: block;
+		height: 1px;
+		margin-top: 5px;
+		position: relative;
+		width: 25px;
 	}
 
-	.button-panel:focus {
-		align-items: flex-start;
-		color: #4c84ff;
+	.policy-detail-border:last-child {
+		border-bottom-left-radius: 8px;
+		border-bottom-right-radius: 8px;
+		padding-bottom: 50px;
+	}
+
+	.nav-link {
+		align-items: center;
+		background-color: #f3f5f7;
+		border: solid 1px #e7e7ed;
+		border-left-color: transparent;
+		border-right-color: transparent;
+		cursor: pointer;
+		display: flex;
```

Comment:
Isn't possible to do it with bootstrap d-flex and align-items-center?

Reply:
I don't know where it's being used, this is class was created in the other history.